tests/integrations/data/local-area-shielding-advice.csv

```diff
@@ -0,0 +1,8 @@
+ladcode,ladname,shielding_advice,start_date,start_time
```

Comment:
If we're controlling the file can we use a `start_datetime` in ISO8601?

Reply:
👍

Reply:
done !